src/olympia/addons/tests/test_admin.py

```diff
@@ -528,6 +532,63 @@ def test_version_pagination(self):
         assert pq(response.content)('#id_files-0-id')[0].attrib['value'] == (
             str(first_file.id))
 
+    def test_git_extract_action(self):
+        addon1 = addon_factory()
+        addon2 = addon_factory()
+
+        addons = Addon.objects.filter(
+            pk__in=(addon1.pk, addon2.pk))
+        addon_admin = AddonAdmin(Addon, admin.site)
+        request = RequestFactory().get('/')
+        request.user = user_factory()
+        core.set_user(request.user)
+        request._messages = default_messages_storage(request)
```

Comment:
so what's this for?  we're not checking activity log or the message output.  (should we be?  `message_user` is being called)

Reply:
I took this from https://github.com/mozilla/addons-server/blob/master/src/olympia/users/tests/test_admin.py#L269:L269

Reply:
that test _does_ check the activity log.  (And should check the message output really, if it's not it's been overlooked)

Reply:
There is no immediate activity log for this action. Should I remove this line then?

Reply:
The test fails without that line.

Reply:
I would and see if it's needed.  It might fail, but then we know it's doing something important.